doc/modules/tree.rst

```diff
@@ -347,7 +344,7 @@ Tips on practical use
     class to the same value. Also note that weight-based pre-pruning criteria,
     such as ``min_weight_fraction_leaf``, will then be less biased toward
     dominant classes than criteria that are not aware of the sample weights,
-    like ``min_samples_leaf``.
+    like ``min_samples_split``.
```

Comment:
My understanding leads me to believe this change is grammatical, but please check?